eclipse-sirius/sirius-components
# Support for delete edge from diagram tool

### Type of this PR 


### Issue(s)
 
 #119

### What does this PR do?

Call the delete handler on an edge when the remove tool is triggered.

### Screenshot/screencast of this PR

![cap 114](https://user-images.githubusercontent.com/1506357/100630360-be57cc00-332a-11eb-88b6-dbdead59a5da.png)

### Potential side effects

* Diagram refresh
* Delete tool

### How to test this PR?

- [ ] Frontend Unit tests
- [ ] Backend Unit tests
* A test case that delete all edges in Robot topography diagram
* On Robot sample, 
  * create a topography diagram
  * select an edge
  * use the palette delete tool 
    * **we expect a diagram refresh without this edge**
  * select another edge
  * use the [SUPPR] keyboard
    * **we expect a diagram refresh without this edge**

### Checklist

- [x] New React components are available in storybook
